### PR TITLE
[fix bug 1374428] Password Manager feature page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Add a new feature page for Password Manager at `/firefox/features/password-manager/`

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1374428

## Testing
Demo: https://bedrock-demo-agibson.us-west.moz.works/en-US/firefox/features/password-manager/

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
@craigcook r? This page is one of those pages that needs to be on prod in < 3 days. Still waiting on final copy checks, but would be good to review this asap.

**Comment by alexgibson** [comment]
Ok, this page has been approved as well - good to merge once reviewed.

**Comment by alexgibson** [comment]
OK, some copy updates are still coming for this one. do-not-merge again for now.

File: bedrock/firefox/templates/firefox/features/password-manager.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          a <a href="{{ url }}">master password</a>.
+        {% endtrans %}
+        </p>
+      </li>
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/password-manager/firefox-account.png') }}" alt="">
+        <h3>{{ _('Password ninja') }}</h3>
+        <p>
+        {% trans %}
+          No more ‘try again’ while you’re trying to get somewhere. Log in to your Firefox Account
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/password-manager.html]: Nit: should probably be "try again" in double-quotes for US English.

**Comment by flodolo** [comment]
> OK, some copy updates are still coming for this one. do-not-merge again for now.

Was this still supposed to not be merged, or that's just a left-over tag?

**Comment by alexgibson** [comment]
Rebased and updated - this one should be good to go 👍

**Comment by alexgibson** [comment]
Ok, merging this as was approved and given r+wc

**Pull request merged**